ui/src/saros/stf/server/rmi/htmlbot/widget/impl/RemoteHTMLView.java

```diff
@@ -179,19 +176,6 @@ private void ensureExistence(ISelector selector) throws RemoteException {
   }
 
   private boolean exists(ISelector selector) {
-    boolean foundIt = false;
-    try {
-      foundIt =
-          getBrowser()
-              .containsElement(selector)
-              .get(BotPreferences.SHORT_TIMEOUT, TimeUnit.MILLISECONDS);
-    } catch (InterruptedException e) {
-      Thread.currentThread().interrupt();
-    } catch (ExecutionException e) {
-      log.error("could not determine whether element exists", e);
-    } catch (TimeoutException e) {
-      log.error("could not determine whether element exists", e);
-    }
-    return foundIt;
+    return new JQueryHelper(getBrowser()).selectionExists(selector);
```

Comment:
Use `jQueryHelper` object held in instance variable of super class `HTMLSTFRemoteObject` instead?

Reply:
Ok, this seems to be a problem in my understanding of the class. I assumed that `getBrowser()` would just return the browser object held by this object, in which case this could be replaced with the usage of the `jQueryHelper` held by the object. But, in actuality, this call gets a specific browser object from the `browserManager` (see [here](https://github.com/saros-project/saros/blob/b4f129c972ade993cb4307d324bee7bcbb37d8a9/ui/src/saros/stf/server/rmi/htmlbot/widget/impl/RemoteHTMLView.java#L170)).

This could be avoided by adjusting the name of the method to better reflect the functionality. But this does not have to be done as part of this PR.

Reply:
In my first version I always created a new instance of JQueryHelper. I understood [the point of @AbdBarho](https://github.com/saros-project/saros/pull/480#discussion_r269471682) and just created a new instance if the browser changes. If you like I can change it. I see no big improvement in readability.